AmplifyPlugins/Auth/AWSAuthPlugin/Support/Constants/AuthPluginErrorConstants.swift

```diff
@@ -74,6 +74,31 @@ extension AuthPluginErrorConstants {
         "challengeResponse is required to confirmSignIn",
         "Make sure that a valid challenge response is passed for confirmSignIn"
     )
+
+    static let confirmResetPasswordUsernameError: AuthPluginValidationErrorString = (
+        "username",
+        "username is required to confirmResetPassword",
+        "Make sure that a valid username is passed for confirmResetPassword"
+    )
+
+    static let confirmResetPasswordError: AuthPluginValidationErrorString = (
```

Comment:
For consistency, Seems like this variable name should be `confirmResetPasswordNewPasswordError`

Reply:
Changed